packages/fast-components-class-name-contracts-base/src/stack-panel/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * The class name contract for the stackpanel component
+ */
+export interface StackPanelClassNameContract {
+    /**
+     * The root of the stackpanel component
+     */
+    stackPanel?: string;
+
+    /**
+     * the container that contains the stack panel items
+     */
+    stackPanel_itemContainer?: string;
```

Comment:
I would prefer to just say items, so `stackPanel_items`.

Reply:
sure